pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -46,7 +47,7 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     # This is an unfortunate name, but it's actually how to build a release
     # binary for BuckleScript
-    node scripts/install.js
+    npm run postinstall
```

Comment:
:+1: I suppose this is more maintainable.

Reply:
yeah essentially now it's the same thing but in case the location of script or something changes the npm script name will stay the same. That's being said if something like that happens our prePatch phase would start failing so this doesn't help much.